### DISCUSSION TITLE
How to call function after model in input changed?

### DISCUSSION THREAD

**Discussion by bubnenkoff** [alpinejs/alpine#2609] [1. Help] [answered]
I need to call function after value in input changed. I tried to do it with x-click, but problem that click is works _before_ model changed to true. I need to get it work _after_ model is changed.

https://codepen.io/bubnenkoff/pen/jOaNyeZ

**Comment by SimoTod** [comment, marked as answer]
You can use `@change` or `x-effect="console.log(foo)"`
